### DISCUSSION TITLE
Pending change to app text

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3504] [General]
![image](https://github.com/corona-warn-app/cwa-website/assets/66998419/9498f03c-4039-4192-9b10-f1527c932bd6)

**Comment by MikeMcC399** [comment]
I have deleted the app from my personal mobile device as I do not need any of the remaining functions. I will keep it on a test device only now.

**Comment by Ein-Tim** [comment]
Are you expecting any answer / information on this discussion thread? Or was this just a "Show & Tell"?

**Reply by MikeMcC399** [reply]
I was not asking for any specific response. You are of course welcome to comment.

If the RC is released then the version number should be updated in the web, although the section where it appears "This is how the app works best" is outdated, since it assumes that the app is still actively working with exposure notifications, so the version number is no longer relevant for that aspect.
